### DISCUSSION TITLE
Developer section in pom

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#547] [General]
We have a developer section in the pom that seems outdated/incomplete:

```
<developers>
<developer>
	<name>Igor Fedorenko</name>
	<organization>Sonatype</organization>
	<organizationUrl>http://www.sonatype.com</organizationUrl>
</developer>
<developer>
	<name>Tobias Oberlies</name>
	<organization>SAP</organization>
	<organizationUrl>http://www.sap.com</organizationUrl>
</developer>
<developer>
	<name>Jan Sievers</name>
	<organization>SAP</organization>
	<organizationUrl>http://www.sap.com</organizationUrl>
</developer>
</developers>
```

If we want to keep this, could it be automatic updated? Should we list every contributor (also historic?) and should we update as part of committer election?

**Comment by akurtakov** [comment]
As per https://maven.apache.org/pom.html#Developers "A good rule of thumb is, if the person should not be contacted about the project, they do not need to be listed here" so committers only.

**Comment by akurtakov** [comment]
What do you think about simply pointing to https://projects.eclipse.org/projects/technology.tycho/who without listing people explicitly?

**Reply by mickaelistria** [reply]
or https://github.com/eclipse-tycho/tycho/graphs/contributors
